From: chrysn <chrysn@fsfe.org>
Date: Wed, 20 Feb 2013 15:04:38 +0100
Subject: Allow building even with resource absent from +dfsg package

This removes all references to the flattr.png icon which has been
removed to create the DFSG tarball for reasons outlined in
debian/copyright.
---
 openscad.qrc         | 1 -
 src/AboutDialog.h    | 2 --
 src/AboutDialog.html | 2 +-
 3 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/openscad.qrc b/openscad.qrc
index 8f72f87..f2dd971 100644
--- a/openscad.qrc
+++ b/openscad.qrc
@@ -29,7 +29,6 @@
     <file>icons/prefs3DView.png</file>
     <file>icons/prefsEditor.png</file>
     <file>icons/prefsUpdate.png</file>
-    <file>icons/flattr.png</file>
     <file>src/AboutDialog.html</file>
     <file>images/export.png</file>
     <file>images/axes.png</file>
diff --git a/src/AboutDialog.h b/src/AboutDialog.h
index 5b8be81..6c718a3 100644
--- a/src/AboutDialog.h
+++ b/src/AboutDialog.h
@@ -11,8 +11,6 @@ public:
 	AboutDialog(QWidget *) {
 		setupUi(this);
 		this->setWindowTitle( QString(_("About OpenSCAD")) + " " + openscad_shortversionnumber.c_str());
-		QUrl flattr_qurl(":icons/flattr.png" );
-		this->aboutText->loadResource( QTextDocument::ImageResource, flattr_qurl );
 		QString tmp = this->aboutText->toHtml();
 		tmp.replace("__VERSION__", openscad_detailedversionnumber.c_str());
 		this->aboutText->setHtml(tmp);
diff --git a/src/AboutDialog.html b/src/AboutDialog.html
index accff27..0738f5d 100644
--- a/src/AboutDialog.html
+++ b/src/AboutDialog.html
@@ -15,7 +15,7 @@
 <body style="font-family:'Arial'; font-size:13pt;">
 
 <p>
-<a align=right href="https://flattr.com/submit/auto?user_id=openscad&amp;url=http://openscad.org&amp;title=OpenSCAD&amp;language=&amp;tags=github&amp;category=software"><img align=right src=":icons/flattr.png" /></a>
+<a align=right href="https://flattr.com/submit/auto?user_id=openscad&amp;url=http://openscad.org&amp;title=OpenSCAD&amp;language=&amp;tags=github&amp;category=software">Flattr this!</a>
 </p>
 
 <p>
